closed/autoconf/custom-hook.m4

```diff
@@ -59,11 +59,14 @@ AC_DEFUN([OPENJ9_CONFIGURE_CMAKE],
       fi
     ],
     [
-      if test "x$OPENJDK_BUILD_OS" = xmacosx ; then
-        with_cmake=cmake
-      else
-        with_cmake=no
-      fi
+      case "$OPENJ9_PLATFORM_CODE" in
+        xa64|oa64)
```

Comment:
Please put these in alphabetical order (`oa64|xa64`) like on [line 241](https://github.com/dnakamura/openj9-openjdk-jdk11/blob/cmake_xlinux/closed/autoconf/custom-hook.m4#L241).